var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -218,24 +218,32 @@ def nvariant_bool(feature):
         includes  = variant_bool('+plugins')
 
         cmake_args = [
-            '-DPARAVIEW_BUILD_QT_GUI:BOOL=%s' % variant_bool('+qt'),
             '-DVTK_OPENGL_HAS_OSMESA:BOOL=%s' % variant_bool('+osmesa'),
             '-DVTK_USE_X:BOOL=%s' % nvariant_bool('+osmesa'),
             '-DVTK_RENDERING_BACKEND:STRING=%s' % rendering,
             '-DPARAVIEW_INSTALL_DEVELOPMENT_FILES:BOOL=%s' % includes,
             '-DBUILD_TESTING:BOOL=OFF',
             '-DBUILD_EXAMPLES:BOOL=%s' % variant_bool('+examples'),
-            '-DVTK_USE_SYSTEM_FREETYPE:BOOL=ON',
-            '-DVTK_USE_SYSTEM_HDF5:BOOL=%s' % variant_bool('+hdf5'),
-            '-DVTK_USE_SYSTEM_JPEG:BOOL=ON',
-            '-DVTK_USE_SYSTEM_LIBXML2:BOOL=ON',
-            '-DVTK_USE_SYSTEM_NETCDF:BOOL=ON',
-            '-DVTK_USE_SYSTEM_EXPAT:BOOL=ON',
-            '-DVTK_USE_SYSTEM_TIFF:BOOL=ON',
-            '-DVTK_USE_SYSTEM_ZLIB:BOOL=ON',
-            '-DVTK_USE_SYSTEM_PNG:BOOL=ON',
-            '-DOpenGL_GL_PREFERENCE:STRING=LEGACY'
-        ]
+            '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
+
+        if spec.satisfies('@5.8:'):
+            cmake_args.extend([
+                '-DPARAVIEW_USE_QT:BOOL=%s' % variant_bool('+qt'),
+                '-DVTK_USE_EXTERNAL=ON',
```

Comment:
is this right? it does not seem to be in master.

Reply:
It's a bit off.  It should be `-DPARAVIEW_BUILD_WITH_EXTERNAL=ON`.  I'm testing it now, although it reveals a few missing dependencies that need addressing.  I'm moving this to draft until I can finish vetting it.